rotkehlchen/api/v1/resources.py

```diff
@@ -714,6 +714,12 @@ def get(self) -> Response:
         return self.rest_api.version_check()
 
 
+class PingResource(BaseResource):
+
+    def get(self) -> Response:
```

Comment:
The backend part looks fine. Can you also add a test though? It should go here: https://github.com/rotki/rotki/blob/develop/rotkehlchen/tests/api/test_misc.py

Just test that querying this endpoint returns `True` and an empty message